examples/apochromatic/input_apochromatic_pl.in

```diff
@@ -35,43 +35,43 @@ dr2.ds = 2.0
 
 q1.type = plasma_lens_chromatic
 q1.ds = 0.331817852986604588
-q1.k = 996.147787384348956
+q1.k = -996.147787384348956
```

Comment:
Maybe I am just confused, byt why are all the lenses now set to defocusing (or are they)?

Reply:
But the unit was not changed... so, was this a bug before the PR?